lib/index.d.ts

```diff
@@ -2430,6 +2430,17 @@ interface Recognizer {
      */
     getNode(context: ParserContext): (this: ParserContext) => CssNode;
 
+    /**
+     * Any additional methods that return a node or list when called.
+     *
+     * @param context - The parsing context.
+     * @returns A function that produces either a CssNode or List<CssNode>.
+     */
+    [methodName: string]:
+        ((context: ParserContext) => ((this: ParserContext) => CssNode | List<CssNode>)) |
+        ((this: ParserContext, ...args: unknown[]) => CssNode | List<CssNode>) |
+        undefined;
```

Comment:
Not quite sure if this is the correct way to do this. Custom methods are mostly the second signature but I had to add the first to account for `getNode()`.